lib/board.rb

```diff
@@ -34,8 +34,48 @@ def free_positions
     board.flatten.select { |cell| cell.is_a?(Integer) }
   end
 
+  def string_to_board(board_string)
+    rows = board_string.split(",")
+
+    board.size.times do |index_row|
+      board.size.times do |index_column|
+        if rows[index_row].chars[index_column] == Mark::CROSS
+          @counter += 1
+          board[index_row][index_column] = Mark::CROSS
+        elsif rows[index_row].chars[index_column] == Mark::ROUND
+          @counter += 1
+          board[index_row][index_column] = Mark::ROUND
+        end
+      end
+    end
+  end
+
+  def to_s
```

Comment:
I like the idea to use the to_s for logs.

I guess I could moved that into the helper as well